topics/transcriptomics/tutorials/scrna-case_monocle3-trajectories/tutorial.md

```diff
@@ -0,0 +1,687 @@
+---
+layout: tutorial_hands_on
+
+title: 'Trajectory Analysis using Monocle3 '
+subtopic: single-cell-CS
+priority: 5
+zenodo_link: 'https://zenodo.org/record/7078524'
+
+questions:
+- How to prepare input files for Monocle starting from AnnData object?
+- How can I infer lineage relationships between clusters, without a time series?
+- What trajectory analysis can tell us?
+objectives:
+- Understand which opertions to perform on AnnData object to obtain files needed for Monocle
+- Follow the Monocle3 workflow and choose the right parameter values
+- Compare output from Scanpy and Monocle
+- Interpet trajectory analysis results
+
+time_estimation: 1H
+
+key_points:
+- You should understand your data to be able to extract relevant information for further analysis.
+- Trajectory analysis is highly dependent on the parameter values you choose, as such ‘inferred relationships’ are a bigger mathematical leap. Therefore, you should always check if the output makes biological sense before processing to the next step.
+- Comparing the output of two different methods applied on the same dataset might be useful to confirm the results, ensure that the findings are reliable and even sometimes find a new piece of information.
+
+requirements:
+-
+    type: "internal"
+    topic_name: transcriptomics
+    tutorials:
+        - scrna-case_alevin
+        - scrna-case_alevin-combine-datasets
+        - scrna-case_basic-pipeline
+        - scrna-case_JUPYTER-trajectories
+tags:
+- single-cell
+- trajectory-analysis
+- paper-replication
+
+contributors:
+- wee-snufkin
+- nomadscientist
```

Comment:
If you like can use our fancy new contributions syntax to more accurately record contributions to a training material, declaring which people should be recorded for authorship or just editing.

```
contributions:
  authorship:
    - shiltemann
  editing:
    - bebatut
    - hexylena
  funding:
    - carpentries
  testing:
    - userX
  ux:
    - userY
  infrastructure:
    - userZ
```

see the [full gtn tutorial schema documentation](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/schemas/tutorial.html) for details on the individual roles.